hikari/utilities/routes.py

```diff
@@ -291,13 +291,17 @@ def compile_to_file(
 PATCH_CHANNEL: typing.Final[Route] = Route(PATCH, "/channels/{channel}")
 DELETE_CHANNEL: typing.Final[Route] = Route(DELETE, "/channels/{channel}")
 
+POST_CHANNEL_FOLLOWERS: typing.Final[Route] = Route(POST, "/channels/{channel}/followers")
```

Comment:
I assume there is a `GET` for this or the crosspost one, since Discord tells me how many people I am making announcements to when I make them.

Reply:
it's `GET channels/{channel.id}/follower-stats` and bots can't access it